allianceutils/rules.py

```diff
@@ -24,7 +24,7 @@ def has_perms(perms, obj=None):
     :param perms: permissions to check
     :return: django_rules predicate
     """
-    name = 'has_perms:%s' % ','.join(perms)
+    name = 'has_perms:' + ','.join(perms)
```

Comment:
just curious, why the + here? 
i'd thought % would still be better if you somehow dont want `f{','.join(perms)}`

Reply:
I actually wrote it with an f-string but with the nested quotes it looked messy and seemed a bit clearer this way

`'has_perms:' + ','.join(perms)`
vs
`f'has_perms:{','.join(perms)}'`